### DISCUSSION TITLE
Joomla\CMS\Installer\Installer\getParams ignores multiple=true setting

### DISCUSSION THREAD

**Discussion by brbrbr** [joomla/joomla-cms#47036] [2. Cold Features]
### Steps to reproduce the issue

The function getParams always initializes the values as 'string', for multiselect values this should be an array with the expanded values

**Comment by chmst** [comment]
Could you give more context?

**Comment by brbrbr** [comment]
given for example a config field in an extension xml file like:

` <field multiple="true" default="1,2" ....>
          <option value="1">1</option>
          <option value="2">2</option>
          <option value="3">3</option>
        </field>
` 
The installation will set the default value to be a string in: 
https://github.com/joomla/joomla-cms/blob/22f8289d039c752409854e94d3fce75caf335be6/libraries/src/Installer/Installer.php#L1746C14-L1746C56
 while for a multiple=true field it should be an array (explode on , from the string).

As a result, the extension will not use the desired default values.
